internal/backend/temporalclient/client.go

```diff
@@ -119,6 +122,36 @@ func (c *impl) startDevServer(ctx context.Context, cfg *Config, opts client.Opti
 
 func (c *impl) Temporal() client.Client { return c.client }
 
+func (c *impl) TemporalAddr() (frontend, ui string) {
+	if c.srv == nil {
+		if frontend = c.cfg.HostPort; frontend == "" || frontend == "localhost" {
+			// known temporal defaults.
+			frontend = "localhost:7233"
+			ui = fmt.Sprintf("http://%s:%d", "localhost", 8233)
+		}
+		return
+	}
+
+	frontend = c.srv.FrontendHostPort()
+
+	host, port, err := net.SplitHostPort(frontend)
+	if err != nil {
+		return
+	}
+
+	nport, err := strconv.Atoi(port)
+	if err != nil {
+		return
```

Comment:
you just hide the error here?

Reply:
there isn't much i can do here. this will cause the UI addr to be empty and not displayed.

Reply:
Is it user config based error? If so I prefer to fail and let him know something went wrong rather than ignoring it's config

Reply:
tbh there is no way this will fail - it comes from the frontend address temporal returns. not worth the effort doing anything there. i could just put it down as "unknown".